lib/broadside/deploy/ecs_deploy.rb

```diff
@@ -73,6 +73,20 @@ def bootstrap
         info "Service '#{family}' doesn't exist, creating..."
         EcsManager.create_service(config.ecs.cluster, family, @deploy_config.service_config)
       end
+
+      run_bootstrap
+    end
+
+    def run_bootstrap
```

Comment:
we don't _need_ to call `deregister_last_n_tasks_definitions` but it does leave the ECS history a little less cluttered.

i think however that just putting it in the `bootstrap` method inline is that without `update_task_revision` i don't think the service and task_definition are linked yet... so you do at least need that, which prior to now was not part of `bootstrap`.

doing basically what you did in `run_bootstrap` at the end of `bootstrap` and then deregistering means you will leave the config in the same place the old `bootstrap` did... doesn't _really_ matter but it's not the worst idea either.